jobs/env/ci-kubernetes-e2e-gci-gce-ipvs.env

```diff
@@ -0,0 +1,10 @@
+### job-env
+# Use proxy mode IPVS
+KUBE_FEATURE_GATES=SupportIPVSProxyMode=true
+KUBEPROXY_TEST_ARGS=--proxy-mode=ipvs
+
+# Enable the PodSecurityPolicy in tests.
+ENABLE_POD_SECURITY_POLICY=true
+
+CREATE_CUSTOM_NETWORK=true
```

Comment:
We don't need to create custom network in CIs.

Reply:
actually, I was referenced `pull-kubernetes-e2e-gce` :( , thanks for reminding.